solutions/src/main/java/ru/sberbank/school/task02/ClientController.java

```diff
@@ -0,0 +1,111 @@
+package ru.sberbank.school.task02;
+
+import lombok.NonNull;
+import org.apache.commons.cli.CommandLine;
+import ru.sberbank.school.task02.exception.*;
+import ru.sberbank.school.task02.util.*;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ClientController implements FxClientController {
+
+    private FxConversionService calculator;
+    private ArgumentParser parser;
+
+    public ClientController(FxConversionService calculator) {
+        this.calculator = calculator;
+        this.parser = new ArgumentParser();
+    }
+
+    @Override
+    public List<FxResponse> fetchResult(List<FxRequest> requests) {
+
+        List<FxResponse> responses = new ArrayList<>();
+
+        for (FxRequest request : requests) {
+            responses.add(fetchResult(request));
+        }
+
+        return responses;
+    }
+
+    @Override
+    public FxResponse fetchResult(FxRequest requests) {
+
+        if (requests == null) {
+            throw new ConverterConfigurationException("invalid request");
+        }
+
+        String symbol = checkSymbol(requests.getSymbol());
+        String direction = checkDirection(requests.getDirection());
+        String amount = checkDecimal(requests.getAmount());
+        String date = LocalDateTime.now().toString();
+
+        BigDecimal price = calculator.convert(getDirection(direction),
+                                              getSymbol(symbol),
+                                          new BigDecimal(amount));
+
+        return price == null ?
```

Comment:
&#39;?&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a803912836>*